main/py-tz/APKBUILD

```
@@ -2,16 +2,16 @@
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=py-tz
 _pkgname=pytz
-pkgver=2018.9
-pkgrel=1
+pkgver=2019.1
+pkgrel=0
 pkgdesc="A Python definitions of world timezone"
 url="http://pytz.sourceforge.net/"
 arch="noarch"
 license="MIT"
 makedepends="py-setuptools py3-setuptools"
 subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
 source="https://pypi.io/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
-builddir="$srcdir/$_pkgname-$pkgver"
+builddir="$srcdir/$_pkgname-release_$pkgver/src"
```

What made you change this? It is breaking your build.

```suggestion
builddir="$srcdir/$_pkgname-$pkgver/src"
```

---

I had a very strange issue with source file's checksum. Old archive was using different directory so I had to adjust it this way. For some reason it was partially working with new file. Implementing your kind suggestion makes this package build just fine. Thank you!